### PR TITLE
propeperly handle sd_event_add_signal() errors

### PR DESCRIPTION
**Pull request by poettering**

### CODE REVIEW

**Comment by evverx** [comment]
This pull request **introduces 1 alert** when merging 9d99418515fe43eeba202f7076348ed91900d15b into 429926e9cc7f5f7cfd5c2ecaee2ce127c324b47d - [view on LGTM.com](https://lgtm.com/projects/g/systemd/systemd/rev/pr-8a5843d99edd1c880d50041d1378b908c4f7ffdc)

**new alerts:**

* 1 for Dead code due to goto or break statement

---

*Comment posted by [LGTM.com](https://lgtm.com)*

File: src/hostname/hostnamed.c [hunk 1]
```diff
@@ -733,11 +733,14 @@ int main(int argc, char *argv[]) {
 
         r = sd_event_add_signal(event, NULL, SIGINT, NULL, NULL);
         if (r < 0)
-                return r;
+                log_error_errno(r, "Failed to install SIGINT handler: %m");
+                goto finish;
```
Comment (yuwata) [inline src/hostname/hostnamed.c:737]: Oh, LGTM analysis is correct. Missing `{}`.

Comment (poettering) [reply src/hostname/hostnamed.c:737]: ouch!

**Comment by poettering** [comment]
force pushed a new version that fixes lgtm's complaint. ptal

**Pull request merged**